src/plugins/chrome.rs

```diff
@@ -85,10 +85,10 @@ pub fn cookies() -> Result<Ident, crate::Error> {
     }
 
     Ok(Ident {
-        csrf: m.get("csrftoken").ok_or(Error::NoneError)?.to_string(),
+        csrf: m.get("csrftoken").ok_or(Error::ChromeNotLogin)?.to_string(),
         session: m
             .get("LEETCODE_SESSION")
-            .ok_or(Error::NoneError)?
+            .ok_or(Error::ChromeNotLogin)?
```

Comment:
Something is strange, as i see in https://github.com/clearloop/leetcode-cli/blob/master/src/plugins/chrome.rs:88, it's already ChromeNotLogin, but this diff show. So I have messed up too. And scenario like this exist in the PR. FYI @manank20